crates/router/src/types/api.rs

```diff
@@ -65,6 +65,13 @@ pub trait ConnectorCommon {
         "application/json"
     }
 
+    fn validate_auth_type(
+        &self,
+        _val: &types::ConnectorAuthType,
+    ) -> Result<(), error_stack::Report<errors::ConnectorError>> {
+        Ok(())
```

Comment:
Let's have only function signature here, So that we can enforce that this method is being implemented for any connectors which is implementing ConnectorCommon trait